service/scaladsl/server/src/main/scala/com/lightbend/lagom/scaladsl/server/LagomServer.scala

```diff
@@ -77,11 +77,12 @@ object LagomServer {
 
 trait LagomServerComponents extends MetricsServiceComponents {
   def httpConfiguration: HttpConfiguration
+  def playBodyParsers: PlayBodyParsers
```

Comment:
this will probably make MiMa checks to fail

Reply:
Yes. 

https://travis-ci.org/lagom/lagom/jobs/396406981#L706-L710

:-)

Will add a new commit to add the proper filters.

Reply:
Done.